src/lib/middleware/api-token-middleware.ts

```diff
@@ -45,21 +45,25 @@ const apiAccessMiddleware = (
         }
 
         try {
-            const apiToken = req.header('authorization');
-            const apiUser = apiTokenService.getUserForToken(apiToken);
-            const { CLIENT, FRONTEND } = ApiTokenType;
+            const apiToken = req.header('authorization') as string;
```

Comment:
Might not need this.
```suggestion
            const apiToken = req.header('authorization');
```